autogen/retrieve_utils.py

```diff
@@ -299,3 +285,71 @@ def query_vector_db(
         where_document={"$contains": search_string} if search_string else None,  # optional filter
     )
     return results
+
+def create_lancedb_from_dir(dir_path, max_tokens, db_path, table_name, chunk_mode,
+                            must_break_at_empty_line, embedding_model_name):
```

Comment:
Could you sync the parameter name in different functions? All use `embedding_model_name` or `embedding_model`.